scripts/standardising.py

```diff
@@ -189,5 +192,18 @@ def standardising(
         # Need GDAL to write to temporary location so no broken files end up in the done folder.
         run_gdal(command, input_file=input_file, output_file=standardized_working_path)
 
+        # Create footprint GeoJSON
+        run_gdal(
+            ["gdal_footprint", "-t_cs", "georef", "-t_srs", "+proj=longlat +ellps=WGS84"],
```

Comment:
`t_cs` [target coordinate system](https://gdal.org/programs/gdal_footprint.html#cmdoption-gdal_footprint-t_cs) I thought I would have to use that if we were using `WSG84`

> why is -t_srs a proj string rather than a epsg code?

Is that not how we should specify `WSG84`?